conf/docklet.conf.template

```diff
@@ -135,10 +135,11 @@
 # default: ""
 # ADMIN_EMAIL_ADDRESS=""
 
-# GLUSTER_VOLUME_QUOTA : whether enable the quota of glusterfs volume or not
+# DATA_QUOTA : whether enable the quota of data or not
 # default: "NO"
```

Comment:
# DATA_QUOTA : whether enable the quota of data volume or not
# True or False, default: False
# DATA_QUOTA=False